packages/nimbus/src/components/tooltip/tooltip-dev.mdx

```diff
@@ -0,0 +1,188 @@
+---
+id: Components-tooltip-dev
+title: Tooltip Engineering Guidelines
+description: Engineering guideline for using the Nimbus Tooltip component
+documentState: InitialDraft
+order: 999
+menu:
+  - Components
+  - Feedback
+  - TooltipDev
+tags:
+  - component
+  - engineering
+  - migration
+---
+
+# Tooltip Component
+
+Tooltips is a compound component that displays informative text when users hover over or focus on an element.
+
+## Component Anatomy [Optional for simple components]
+
+Component Anatomy here - image with parts and description as can be seen on [combobox](https://www.figma.com/design/AvtPX6g7OGGCRvNlatGOIY/NIMBUS-design-system?node-id=5311-13804&p=f&m=dev) for example
+
+![](/images/tooltip/tooltip.png)
+
+## Component Usage
+
+### How to Import
+
+```tsx
+import { Tooltip } from '@commercetools/nimbus';
+```
+
+### Basic Example
+
+A minimal, copy-and-paste example of the tooltip in its simplest form: edit the placement in code code interface to different positions of choice (top, left, right, etc) 
+
+```jsx-live-dev
+// Example: A basic tooltip
+const App = () => (
+<Tooltip.Root placement="top">
+  <Button>Hover me</Button>
+    <Tooltip.Content
+      placement="top"
+      maxWidth="300px"
+      backgroundColor="blue"
+      color="white"
+    >
+      This is a helpful tooltip
+    </Tooltip.Content>
+</Tooltip.Root>
+)
+```
+
+### Common props and Variations
+
+#### Placement Options
+Control where the tooltip appears relative to its trigger: change the placement to any of these options to reposition the tooltip:
+```"top" | "top left" | "top right" | "top start" | "top end" | "bottom" | "bottom left" | "bottom right" | "bottom start" | "bottom end" | "left" | "left top" | "left bottom" | "right" | "right top" | "right bottom" | "start" | "start top" | "start bottom" | "end" | "end top" | "end bottom"```
+
+```jsx-live-dev
+const App = () => (
+<Tooltip.Root>
+      <Button>Tooltip placement</Button>
+      <Tooltip.Content placement="top">Tooltip on top</Tooltip.Content>
+</Tooltip.Root>
+)
+```
+
+#### Offset Options
+
+Control the tooltip offset:
+```offset```
+
+```jsx-live-dev
+const App = () => (
+<Tooltip.Root>
+      <Button>Tooltip placement</Button>
+      <Tooltip.Content offset={50}>Tooltip on top</Tooltip.Content>
+</Tooltip.Root>
+)
+```
+
+```....Add example snippets for each relevant component prop```
+
+### API Changes
```

Comment:
This section feels like it belongs in the migration guide

Reply:
Oh yes, I added it there too. I can take it out from here.

Reply:
I agree, this belongs in the migration guide